tests/unit/spec/base/core-datalayer.js

```diff
@@ -0,0 +1,27 @@
+/* For reference read the Jasmine and Sinon docs
+ * Jasmine docs: http://pivotal.github.io/jasmine/
+ * Sinon docs: http://sinonjs.org/docs/
+ */
+
+/* global describe, beforeEach, afterEach, it, expect, sinon, spyOn, jasmine */
+
+describe('core-datalayer.js', function() {
+
+    describe('PageId', function(){
+
+        it('will grab data-gtm-page-id value if present on <html> element', function(){
+            $('html').attr('data-gtm-page-id', 'test');
```

Comment:
If we're modifying this element in a test, we should clean up after each test so as not to avoid unexpected results

```javascript
afterEach(function () {
    html.removeAttribute('data-gtm-page-id');
});
```

We could also store our reference to this DOM node in a variable for reuse in the test suite:

```javascript
var html = document.documentElement;
```